### PR TITLE
Generalize cspr.live references

### PR DESCRIPTION
**Pull request by GloreenS**
### Related links

Card #136 and #138 

### Changes

- Reworded references to cspr.live
- Updated capitalization of commonly used terms to be consistent with other topics
- Split topic "Transferring Tokens on the Casper Testnet" to generalize transfer instructions and created separate topic for testnet faucet feature "Funding Testnet Accounts"
- Updated references to UI elements in Transferring Tokens to match the actual UI
- Added references to the signer guide and removed steps repeated from the signer guide

#### Latest commits 
- Added a new topic for block explorers
- Updated references of block explorers to point to the block explorers topic

### Notes

Testnet is available only on the cspr.live block explorer, so added a separate reference to this in the Funding Testnet Accounts topic.

### CODE REVIEW

File: source/docs/casper/workflow/signer-guide.md [hunk 1]
```diff
@@ -152,18 +154,18 @@ To view your account information, do the following:
...
 
-## 7. Accessing the CSPR Wallet from cspr.live
+## 7. Accessing the CSPR Wallet from a Block Explorer
 
 Once you are logged in to the Casper Signer, you can access the wallet for each account registered in the Signer. For more information on how to log in to the Signer, see [Logging in to the Casper Signer](signer-guide#12-logging-in-to-the-casper-signer).
 
 Alternatively, you can follow these steps to log in to your Signer/CSPR wallet:
 
-1. Navigate to the CSPR Mainnet https://cspr.live/.
+1. Navigate to the [cspr.live](https://cspr.live/).
```
Comment (Fraser999) [inline source/docs/casper/workflow/signer-guide.md:163]:
```suggestion
1. Navigate to [cspr.live](https://cspr.live/), using Chrome or a Chromium-based browser like Brave.
```

File: source/docs/casper/workflow/staking.md [hunk 2]
```diff
@@ -50,7 +50,7 @@ You can create, store, and use one or more CSPR accounts with your Signer wallet
 
 If you are new or have logged out of the Signer, you can log in with these steps:
 
-1.  Using Chrome or a Chromium-based browser like Brave, navigate to the block explorer on the mainnet (<https://cspr.live/>), and click on the **Sign-in** menu.
+1.  Using Chrome or a Chromium-based browser like Brave, navigate to a block explorer on the Mainnet, for this example we are using (<https://cspr.live/>), and click on the **Sign-in** menu.
```
Comment (Fraser999) [inline source/docs/casper/workflow/staking.md:53]:
```suggestion
1.  Using Chrome or a Chromium-based browser like Brave, navigate to a block explorer on the Mainnet, (for this example we are using [cspr.live](https://cspr.live/)), and click on the **Sign-in** menu.
```

File: source/docs/casper/workflow/ledger-setup.md [hunk 3]
```diff
@@ -2,7 +2,7 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 
 # Ledger Setup
 
-A Ledger Device is a hardware wallet that is considered one of the most secure ways to store your digital assets. Ledger uses an offline, or cold storage, method of generating private keys, making it a preferred method for many crypto users. This guide will help you to connect your Ledger device to the Casper Web wallet (<https://cspr.live>). The Casper Web wallet enables you to send and receive CSPR tokens.
+A Ledger Device is a hardware wallet that is considered one of the most secure ways to store your digital assets. Ledger uses an offline, or cold storage, method of generating private keys, making it a preferred method for many crypto users. This guide will help you to connect your Ledger device to a Casper Web wallet on a block explorer such as (<https://cspr.live>). The Casper Web wallet enables you to send and receive CSPR tokens.
```
Comment (Fraser999) [inline source/docs/casper/workflow/ledger-setup.md:5]:
```suggestion
A Ledger Device is a hardware wallet that is considered one of the most secure ways to store your digital assets. Ledger uses an offline, or cold storage, method of generating private keys, making it a preferred method for many crypto users. This guide will help you to connect your Ledger device to a Casper Web wallet on a block explorer such as [cspr.live](https://cspr.live/). The Casper Web wallet enables you to send and receive CSPR tokens.
```

File: source/docs/casper/workflow/testnet-faucet.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+The Casper Testnet is an alternate Casper blockchain aimed at testing applications running on the Casper Network without spending CSPR tokens on the Casper Mainnet. Testnet tokens are independent and separate from the actual Casper Token (CSPR). While test tokens do not have any monetary value, they possess the same functionality as the CSPR token within the confines of the Testnet. The Testnet is deployed independently from the Casper Mainnet for users to experiment with Casper Network features such as transferring, delegating, and undelegating tokens. You can access the Casper Testnet using https://testnet.cspr.live/. 
+
+**Figure 1**: Casper Testnet Home
+
+<img src={useBaseUrl("/image/workflow/testnet-home.png")} width="500" alt="Casper Testnet Home" />
+
+## 2. The Faucet
```
Comment (Fraser999) [inline source/docs/casper/workflow/testnet-faucet.md:11]: This numbering doesn't make sense does it?  Same for 2.1 below.

**Review by ipopescu — DISMISSED** [review]
The content looks good, thanks @GloreenS. I want to run one question by the team before merging.

File: source/docs/casper/workflow/testnet-faucet.md (hunk 4 above)
Comment (GloreenS) [reply source/docs/casper/workflow/testnet-faucet.md]: Updated in [dabcfd9](https://github.com/casper-network/docs/pull/207/commits/dabcfd92fa82f698fa867ff1b32fc4356310b017)

**Review by ipopescu — APPROVED** [review]
Looks great, @GloreenS! Thank you!

**Pull request merged**